spec/lib/rubocop/cop/no_reset_column_information_migrations_spec.rb

```diff
@@ -8,6 +8,8 @@
 
   let(:config) { RuboCop::Config.new }
 
+  before { config["Discourse/NoResetColumnInformationInMigrations"]["Enabled"] = true }
```

Comment:
This cop is disabled by default (always was it seems) but for whatever reason it was working until now.